src/switch/index.tsx

```diff
@@ -0,0 +1,162 @@
+import { DNode } from '@dojo/framework/core/interfaces';
+import focus from '@dojo/framework/core/middleware/focus';
+import coreTheme from '@dojo/framework/core/middleware/theme';
+import { uuid } from '@dojo/framework/core/util';
+import { create, tsx } from '@dojo/framework/core/vdom';
+
+import { formatAriaProperties } from '../common/util';
+import Label from '../label';
+import * as css from '../theme/default/switch.m.css';
+
+interface SwitchProperties {
+	/** Custom aria attributes */
+	aria?: { [key: string]: string | null };
+	/**  Checked/unchecked property of the switch */
+	checked?: boolean;
```

Comment:
as this is essentially being used as an on/off switch and never a group of checkboxes, how would you feel about changing this to `value`  and removing `checked`